tests/devices/default_qubit/test_default_qubit.py

```diff
@@ -165,7 +172,7 @@ def test_doesnt_support_other_gradient_methods(self, gradient_method):
 class TestBasicCircuit:
     """Tests a basic circuit with one RX gate and two simple expectation values."""
 
-    @pytest.mark.parametrize("max_workers", [None, 1, 2])
+    @pytest.mark.parametrize("max_workers", max_workers_list)
```

Comment:
# Suggestion
Have we considering replacing all of these method-level parameterizations with a single parameterization of the test class?